library/bignum_core.c

```diff
@@ -99,6 +99,7 @@ static mbedtls_mpi_uint mpi_bigendian_to_host(mbedtls_mpi_uint a)
         /* Nothing to do on bigendian systems. */
         return a;
     } else {
+MBEDTLS_IGNORE_UNREACHABLE_BEGIN
```

Comment:
The size of `mbedtls_mpi_uint` is known to the preprocessor so we don't need a workaround on top of a workaround.

```
#if defined(MBEDTLS_HAVE_INT64)
return (mbedtls_mpi_uint) MBEDTLS_BSWAP64(a);
#elif defined(MBEDTLS_HAVE_INT32)
return (mbedtls_mpi_uint) MBEDTLS_BSWAP32(a);
#else
#error "This is one of several places that need to be adapted to support a new limb size"
#endif
```

Reply:
Agreed, this is cleaner. I'll lose the macro.